### PR TITLE
Enhanced notifications for Mac Systems

### PR DESCRIPTION
**Pull request by HappyTobi**
Hi,

i changed the notification tooling to send notifications about important events from gopass. 
When "terminal-notifier" is available gopass will send the notification through the "terminal-notifier" program, else it will use the "oascript" like before.

So i changed the go packager that brew will install the terminal-notifier.

Kind regards

### CODE REVIEW

File: pkg/notify/notify_darwin.go [hunk 1]
```diff
@@ -6,23 +6,55 @@ import (
...
+var execLookPath = exec.LookPath
+
 // Notify displays a desktop notification using osascript
 func Notify(ctx context.Context, subj, msg string) error {
 	if os.Getenv("GOPASS_NO_NOTIFY") != "" || !ctxutil.IsNotifications(ctx) {
 		return nil
 	}
-	osas, err := exec.LookPath("osascript")
+
+	//check if terminal-notifier was installed else use the applescript fallback
```
Comment (dominikschulz) [inline pkg/notify/notify_darwin.go]: please add a space after the comment sign

File: pkg/notify/notify_darwin.go [hunk 2]
```diff
@@ -6,23 +6,55 @@ import (
...
+	tn, _ := executableExists(terminalNotifier)
+	if tn {
+		return tnNotification(msg, subj)
+	}
+	return osaNotification(msg, subj)
+}
+
+func osaNotification(msg string, subj string) error {
```
Comment (dominikschulz) [inline pkg/notify/notify_darwin.go]: Add a short comment, please. Something like: `// display noficiation with osascript`

File: pkg/notify/notify_darwin.go [hunk 3]
```diff
@@ -6,23 +6,55 @@ import (
...
+func execNotification(executable string, args []string) error {
+	return execCommand(executable, strings.Join(args[:], " ")).Start()
+}
 
-	return exec.Command(
-		osas,
-		"-e",
-		`display notification "`+msg+`" with title "`+subj+`"`,
-	).Start()
+func tnNotification(msg string, subj string) error {
```
Comment (dominikschulz) [inline pkg/notify/notify_darwin.go]: Add a short comment here, too.

File: pkg/notify/notify_darwin.go (hunk 1 above)
Comment (HappyTobi) [reply pkg/notify/notify_darwin.go]: done

File: pkg/notify/notify_darwin.go (hunk 2 above)
Comment (HappyTobi) [reply pkg/notify/notify_darwin.go]: i added a short description

File: pkg/notify/notify_darwin.go (hunk 3 above)
Comment (HappyTobi) [reply pkg/notify/notify_darwin.go]: i added a short description

**Pull request merged**